tests/snowflake/test_snowflake.py

```diff
@@ -36,7 +36,7 @@
     domain='test_domain',
     database='test_database',
     warehouse='test_warehouse',
-    query='test_query with %(foo)s and {{ pokemon }}',
+    query='test_query with %(foo)s and %(pokemon)s',
```

Comment:
`%(...)` is an old python syntax, I think we'd prefer to push Jinja' one in our connectors configs
Today, we're already replacing these automatically: https://github.com/ToucanToco/toucan-connectors/blob/ce8ed0622daf262abcec7f72bf0d90f6506c6962/toucan_connectors/common.py#L78

So my suggestion would be to use jinja {{ }} in your tests and documentation as well

Reply:
les syntaxes compatibles avec execute de snowflake sont

* pyformat (`%()s`)  => bug car on passe un dictionnaire parameters qui peut comporter des valeurs invalides.
* qmark => cest dans ce format qu'on convertit notre query pour bypasser le probleme du dictionnaire.

Par contre on ne peut pas utiliser jinja car ce n'est pas safe sur des requetes SQL
cf le code que tu pointes, qui est extrait de la methode no_sql_apply_parameters_to_query :

```
def nosql_apply_parameters_to_query(query, parameters, handle_errors=False):
    """
    WARNING : DO NOT USE THIS WITH VARIANTS OF SQL
    Instead use your client library parameter substitution method.
    https://www.owasp.org/index.php/Query_Parameterization_Cheat_Sheet
    """
```

on veut spécifiquement éviter le templating de cette manière.

Reply:
@davinov you already had an issue on that actually! https://github.com/ToucanToco/toucan-connectors/issues/303
^^